src/components/tab/Tab.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
+
+export interface ITabOwnProps {
+  id: string;
```

Comment:
`id` should be optional. The reason we separate `Tab` and `TabConnected` is so you can use `Tab` on itself and in that case, you do not need the id prop.